docs/reference/01-scnode-api-spec.rst

```diff
@@ -2413,3 +2413,191 @@ ________
 
 ________
 
+.. http:post:: /submitter/getSchnorrPublicKeyHash
```

Comment:
Let's create separate section for key rotation operations
Just put at the beginning following text

=====
**Key Rotation operations**
=====

Reply:
Done